Sources/SymbolKit/SymbolGraph/Symbol/ValueConstraints.swift

```diff
@@ -0,0 +1,132 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+extension SymbolGraph.Symbol {
+    /// Set of constraints that describe the value held by a variable.
+    /// 
+    /// These constraints include the allowed minimum or maximum value,
+    /// the minimum or maximum lengths of a string value,
+    /// the exact allowed values, and the default value assumed when one isn't
+    /// explicitly specified.
```

Comment:
Is this doc comment meant to be attached to anything?

Reply:
This was meant to be a regular comment. Thanks for catching